src/core/tests/opset.cpp

```diff
@@ -71,7 +71,7 @@ INSTANTIATE_TEST_SUITE_P(opset,
                                          OpsetTestParams{ov::get_opset10, 177},
                                          OpsetTestParams{ov::get_opset11, 177},
                                          OpsetTestParams{ov::get_opset12, 178},
-                                         OpsetTestParams{ov::get_opset13, 184}),
+                                         OpsetTestParams{ov::get_opset13, 185}),
```

Comment:
@mitruska, @ilyachur, what's the purpose of this test? Do we have requirements on the number of ops in the opsets?

Reply:
I'm not an author of this test, but as I understand it checks the expected number of the operation in the opset. 
For example to ensure that the new operation has been registered in the opsetN correctly or that no operation has been removed by accident.